Editor/Deployment/DeploymentSettings.cs

```diff
@@ -442,7 +442,7 @@ public async Task DeleteDeployment()
         {
             var stackName = _coreApi.GetStackName(_gameName);
             _coreApi.DeleteStack(CurrentProfile, CurrentRegion, stackName);
-            RefreshCurrentStackInfo();
+            _currentDeploymentId.Set(new DeploymentId(CurrentProfile, CurrentRegion, stackName, _deployers[Scenario].DisplayName));
```

Comment:
I'm not sure I understand why this is required? Is someone resetting _currentDeploymentId? Should it not point to the same stack?

Reply:
https://github.com/aws/amazon-gamelift-plugin-unity/blob/9ef50c7b98e4497c4a59d9632a98b58ad54ab700/Editor/Deployment/DeploymentSettings.cs#L273-L276

The id only tracks the act of deploying, so when it is done waiting, it clears it. I set a deploymentid so the poller will track the deletion

Reply:
yeah i think im confused about the original code, and why we are even clearing the id to begin with. But modifying this behavior seems more dangerous and has scope to break multiple things, so for now i think this seems fine to me.